cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -311,7 +317,8 @@ func Run(c *config.CompletedConfig, stopCh <-chan struct{}) error {
 			})
 	}
 
-	select {}
+	<-stopCh
+	return nil
```

Comment:
Comment in line 178 says

> // Run runs the KubeControllerManagerOptions.  This should never exit.

is ok to exit now?

Reply:
Removed the comment.

If one want it to never exit, they can pass wait.NeverStop as an argument (which we do in line 138).
In tests, we want it to stop actually.